### DISCUSSION TITLE
Welcome to ChainBreaker Discussions!

### DISCUSSION THREAD

**Discussion by S-CODE-pl** [S-CODE-pl/ChainBreaker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Damien0505** [comment]
Hi,

I am really excited about the announcement of this new app.  Previously I had used the Speed app, but since upgrading to Daniel Nilsson's bbs-fw sadly I can no longer use it.

So there are a number if things that sound amazing:
- Compatibility with bbs-fw
- Extended wireless Bluetooth adaptor
- Multiple dashboards
- Firmware setting editor

I would be more than happy to test the app at any stage, as well as the concept of the Bluetooth extender.
While I am not one for coding, hardware and electronics is my thing with experience ranging from that of a home hobbyist up to a radio and broadcast tech back in the 80's and 90's.

Currently I have a BBSHD running @ 52V on a Trek Marlin 5 with the bbs-fw and a DPC18 display (ECO/Sport each with 5 PAS levels).

I look forward to see what can come of this project!

Kind regards,
Damien - Australia

**Comment by kklmm** [comment]
Hi,
you're doing amazing work on releasing a replacement for speeed app which no longer works when using daniels opensource firmware.

any eta when we can expect a beta release? looking forward to use this as soon as possible.
very exciting!

**Reply by S-CODE-pl** [reply]
Hi, We're working hard to publish Android+USB release in 1-2 months, stay in touch!
